jme3-android/src/main/java/com/jme3/app/JmeSurfaceView.java

```diff
@@ -0,0 +1,467 @@
+package com.jme3.app;
```

Comment:
Please add the JME copyright notice and license.

Reply:
> So @stephengold ,
> this :
> 
> ```java
>         if ( legacyApplication != null ) {
> ```
> 
> or this :
> 
> ```java
>         if (legacyApplication != null) {
> ```